docs/detections/rules-ui-create.asciidoc

```diff
@@ -311,7 +315,11 @@ NOTE: Rules that use aggregating queries might create duplicate alerts. This can
 [float]
 [[esql-non-agg-query]]
 ===== Non-aggregating query 
-Non-aggregating queries doesn't use `STATS...BY` functions and doesn't aggregate source event data. Alerts generated by an {esql} rule with a non-aggregating query only contain the fields returned by the query.
+Non-aggregating queries don't use `STATS...BY` functions and don't aggregate source event data. Alerts generated by a non-aggregating query contain source event fields that the query returns, new fields the query creates, and all other fields in the source event document. 
+
+TIP: To improve the visibility of these fields while investigating alerts that the {esql} rule generates, add them to the rule's  <<rule-ui-advanced-params,custom highlighted fields>>. 
+
+NOTE: New fields are fields that don't exist in the query’s source index.
```

Comment:
```suggestion
NOTE: New fields are fields that don't exist in the query’s source index. For example fields created in [STATS..BY command](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-commands.html#esql-stats-by) or  [eval command](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-commands.html#esql-eval), etc
```

Reply:
and for aggregating query - let's have an example with `STATS..BY`, that creates a new field

Reply:
Resolving this so we can continue this discussion on the latest version of this PR.